native/cocos/scene/Skybox.cpp

```diff
@@ -270,6 +270,10 @@ void Skybox::activate() {
             skyboxMaterial->release();
             skyboxMaterial = nullptr;
         });
+        EventDispatcher::addCustomEventListener(EVENT_RESTART_VM, [](const CustomEvent & /*unused*/) {
+            skyboxMaterial->release();
+            skyboxMaterial = nullptr;
+        });
```

Comment:
SkyBox is not deleted when restarting? I think ScriptEngine::cleanup() will delete it.

Reply:
Yes. It seems that ScriptEngine::cleanup() DO NOT handle skybox related resource. And skyboxMaterial refCount is 3 if not release by this logic.

Reply:
The log info shows that skyboxMaterial refCount is unchanged whether called ScriptEngine::cleanup()